README.rst

```diff
@@ -53,27 +53,37 @@ From source:
     python setup.py install
 
 
+Why Ignite?
+===========
+Ignite's high level of abstraction assumes less about the type of network (or networks) that you are training, and we require the user to define the closure to be run in the training and validation loop. This level of abstraction allows for a great deal more of flexibility, such as co-training multiple models (i.e. GANs) and computing/tracking multiple losses and metrics in your training loop.
+
+Ignite also allows for multiple handlers to be attached to events, and a finer granularity of events in the engine loop.
+
+
 Documentation
 =============
 API documentation and an overview of the library can be found `here <https://pytorch.org/ignite/index.html>`_.
 
+
+Structure
+=========
+- **ignite**: Core of the library, contains supervised trainer and evaluator, all of the classic machine learning metrics and a variety of handlers to ease the pain of training neural networks! 
+
+- **ignite.contrib**: Contrib directory contains projects created by Ignite users. Projects vary from TBPTT engine, various Parameter Schedulers, Progress Bar and a metrics module containing over 10 Regression metrics (ignite.contrib.metrics.regression)! 
```

Comment:
How about saying like that ?
```suggestion
- **ignite.contrib**: Contrib directory contains additional modules contributed by Ignite users. Modules vary from TBPTT engine, various optimisation parameter schedulers, logging handlers and a metrics module containing over 10 Regression metrics (`ignite.contrib.metrics.regression <https://github.com/pytorch/ignite/tree/master/ignite/contrib/metrics/regression>`_)! 
```

Reply:
nit: `Contrib` -> `The Contrib`
nit: `over 10 Regression metrics` -> `many regression metrics`

Reply:
Incorporated!